backend/src/main.py

```
@@ -1,32 +1,702 @@
 def health():
+    """Simple health check"""
+    # Check database connection
+    db_healthy = False
+    try:
+        conn = get_db_connection()
+        cursor = conn.cursor()
+        cursor.execute("SELECT COUNT(*) FROM opportunities")
+        opportunity_count = cursor.fetchone()[0]
+        conn.close()
+        db_healthy = True
+    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)